telegram/ext/picklepersistence.py

```diff
@@ -41,17 +43,21 @@ class PicklePersistence(BasePersistence[UD, CD, BD]):
         :class:`PicklePersistence` will try to replace :class:`telegram.Bot` instances by
         :attr:`REPLACED_BOT` and insert the bot set with
         :meth:`telegram.ext.BasePersistence.set_bot` upon loading of the data. This is to ensure
-        that changes to the bot apply to the saved objects, too. If you change the bots token, this
-        may lead to e.g. ``Chat not found`` errors. For the limitations on replacing bots see
-        :meth:`telegram.ext.BasePersistence.replace_bot` and
-        :meth:`telegram.ext.BasePersistence.insert_bot`.
+            that changes to the bot apply to the saved objects, too. If you change the bot's
+            token, this may lead to e.g. ``Chat not found`` errors.
+            For the limitations on replacing bots see:
+                * :meth:`telegram.ext.BasePersistence.replace_bot`
+                * :meth:`telegram.ext.BasePersistence.insert_bot`.
```

Comment:
did you check that this renders correctly? Does the extra indentation not break it?

Reply:
reverted